packages/browser/coretest/__tests__/StitchCustomUserDataIntTests.ts

```diff
@@ -0,0 +1,81 @@
+import { BaseStitchBrowserIntTestHarness } from "mongodb-stitch-browser-testutils";
+import { AnonProviderConfig, MongoConfig, MongoDbRule, MongoDbService, Role, Schema, ServiceResource, Rule, UserpassProviderConfig, UserpassProvider, CustomUserConfigData, StitchFunction } from "mongodb-stitch-core-admin-client";
+import { UserPasswordCredential } from "mongodb-stitch-core-sdk";
+
+const harness = new BaseStitchBrowserIntTestHarness();
+
+beforeAll(() => harness.setup());
+afterAll(() => harness.teardown());
+
+describe("StitchAppClient", () => {
+	const db = "db1"
+	const coll = "coll1"
+	const functionName = "addUserProfile";
+	const user = "stitch@10gen.com";
+	const password = "stitchuser";
+	const colorField = "favoriteColor";
+	const favoriteColor = "blue";
+
+	it("should define custom data", async () => {
```

Comment:
I feel like this test structure and string is a bit off. should there be a nested `describe` like `with custom user data` then you have one `it` block `user should have custom data` then another for like `user should have updated custom data after running refresh`? Instead of having everything in one big block. If you don't want to split this up better you could at least make the `it` string a little more descriptive of what is going on.